components/TheFooter.vue

```diff
@@ -174,7 +174,7 @@ const menuKodadot: Menu[] = [
 
   {
     name: $i18n.t('referralProgram'),
-    url: 'https://hello.kodadot.xyz/knowledgebase/technical/kodadots-programs/artist-referral-program',
+    url: 'https://hello.kodadot.xyz/technical/kodadots-programs/artist-referral-program',
```

Comment:
https://hello.kodadot.xyz/be-part-of-kodadot/kodadots-programs/artist-referral-program